app/src/main/java/com/mainApp/yelp_mini/viewModel/DetailViewModelClass.kt

```diff
@@ -1,90 +1,32 @@
 package com.mainApp.yelp_mini.viewModel
 
-import android.util.Log
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.mainApp.yelp_mini.model.data.YelpRestaurantDetail
 import com.mainApp.yelp_mini.model.data.YelpRestaurantReviews
-import com.mainApp.yelp_mini.model.retroServices.RetroInstance
-import com.mainApp.yelp_mini.model.retroServices.YelpService
-import com.mainApp.yelp_mini.model.util.API_KEY
-import retrofit2.Call
-import retrofit2.Callback
-import retrofit2.Response
+import com.mainApp.yelp_mini.model.repo.RepoClass
 
-private const val TAG = "DetailActivityAPICALL"
+private const val TAG = "DetailViewModelClass"
 
 class DetailViewModelClass : ViewModel() {
+    private val repoClass by lazy { RepoClass() }
+    private var restaurantsDetailList: MutableLiveData<YelpRestaurantDetail?> = MutableLiveData()
+    private var restaurantsReviewList: MutableLiveData<YelpRestaurantReviews?> = MutableLiveData()
 
-    val restaurantsDetailList: MutableLiveData<YelpRestaurantDetail> = MutableLiveData()
-    val restaurantsReviewList: MutableLiveData<YelpRestaurantReviews> = MutableLiveData()
 
-
-    fun getRestaurantsDetailLists(): MutableLiveData<YelpRestaurantDetail> {
+    fun getRestaurantsDetailLists(): MutableLiveData<YelpRestaurantDetail?> {
```

Comment:
Use `LiveData` instead of `MutableLiveData` as return type. Also implement this in all places in this class